.github/workflows/integration-tests.yml

```diff
@@ -1,12 +1,22 @@
 name: Integration tests
 
 on:
-  pull_request:
-    branches:
-     - master
   push:
     branches:
       - master
+      - 'branch-**'
+  pull_request:
+    paths-ignore:
+     - docs/*
+     - examples/*
+     - scripts/*
```

Comment:
`scripts` contains `run_integration_test.sh` - we should run integration tests when it is changed.

Reply:
fixed